docs/tutorials/random.rst

```diff
@@ -53,7 +53,7 @@ random integers up to, but not including, the value of the argument ``N``
 Sometimes you need numbers with a decimal point in them. These are called
 *floating point* numbers and it's possible to generate such a number with the
 ``random.random`` method. This only returns values between ``0.0`` and ``1.0``
-inclusive. If you need larger random floating point numbers add the results
+exclusive. If you need larger random floating point numbers add the results
```

Comment:
It's inclusive of 0.0 but exclusive of 1.0.

Maybe something like:
```
This only returns values between ``0.0`` and ``1.0`` (it includes ``0.0`` but excludes ``1.0``).
```

Reply:
Even better 👍 
Updated in https://github.com/bbcmicrobit/micropython/compare/62bc0e400de6b31602080467b34aca8adf084259..3dc86b44404ce521e00c6e041c4a4e020280e5f8

Reply:
Looks good!